trustregistry/registry/actor.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from fastapi import APIRouter
+from pydantic import BaseModel, validator
+from typing import List
+from slugify import Slugify
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/registry/actor", tags=["actor"])
```

Comment:
Another nitpick, but I think common restfull practice is plural resource notation. So actors

```suggestion
router = APIRouter(prefix="/registry/actors", tags=["actor"])
```

Same for schema

Reply:
yeah sure - well spotted